packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2877,15 +2922,28 @@ Iterable<DiagnosticsNode> _describeRelevantUserCode(Element element) {
     // TODO(chunhtai): should print out all the widgets that are about to cross
     // package boundaries.
     if (debugIsLocalCreationLocation(target)) {
-      nodes.add(
+
+      DiagnosticsNode? devToolsDiagnostic;
+      if (activeDevToolsServerAddress != null && WidgetInspectorService.instance.serviceInfo != null) {
+        final String? devToolsInspectorUri =
+            WidgetInspectorService.instance._devToolsInspectorUriForElement(target);
```

Comment:
Wondering whether  _devToolsInspectorUriForElement should just return null if activeDevToolsServerAddress or WidgetInspectorService.instance.serviceInfo is not set. It seems like an odd implementation detail that _devToolsInspectorUriForElement is only available when those two conditions are met...

Reply:
done.